lib/puppet/provider/reboot/windows.rb

```diff
@@ -148,8 +152,8 @@ def package_installer?
     # 0x00000000 (0)	No pending restart.
     path = 'SOFTWARE\Microsoft\Updates'
     value = reg_value(path, 'UpdateExeVolatile')
-     # this may error if this is not a control flow construct
-    if value and value != 0
+    # this may error if this is not a control flow construct
+    if value && value != 0
```

Comment:
I think I changed one of these at one point and ... things exploded.

Reply:
Yup, my change made sense to me at the time as part of https://github.com/puppetlabs/puppetlabs-reboot/pull/100

It ended up being reverted in https://github.com/puppetlabs/puppetlabs-reboot/pull/104

We'll have to revisit this...